packages/core/admin/admin/src/pages/Marketplace/components/NpmPackageCard.tsx

```diff
@@ -200,15 +198,15 @@ const InstallPluginButton = ({
   // Already installed
   if (isInstalled) {
     return (
-      <Box paddingLeft={4}>
-        <Icon as={Check} marginRight={2} width={12} height={12} color="success600" />
+      <Flex gap={2} paddingLeft={4}>
+        <Check width="1.2rem" height="1.2rem" color="success600" />
```

Comment:
previous values were wrong?

Reply:
we don't have a `12` in the spaces array, so it would have defaulted to `1.2rem` afaik